clusters/hive/hypershift/README.md

```diff
@@ -0,0 +1,26 @@
+This directory includes the manifests, including the generated ones, for deploying HyperShift on the `hive` cluster.
+
+# HyperShift CLI and operator images
+[HyperShift CLI](https://hypershift-docs.netlify.app/getting-started/) which is the utility for installing HyperShift as well as creating and destroying hosted clusters and the HyperShift operator which is the core component running on HyperShift management cluster share the same image. 
+They must have the exact same version to work properly [HOSTEDCP-792](https://issues.redhat.com/browse/HOSTEDCP-792).
+
+The image is built with [hypershift-cli.yaml](clusters/app.ci/supplemental-ci-images/hypershift/hypershift-cli.yaml) to available for all build farms.
+
+# Deployment/upgrading on `hive`
+Prepare the AWS credentials for your management cluster account (in this case, the AWS account running `hive` cluster) in `~/workspace/mgmt-aws` (or any other path you prefer).
```

Comment:
```suggestion
Prepare the AWS credentials for the management cluster account (in this case, the AWS account running `hive` cluster) in `~/workspace/mgmt-aws` (or any other path you prefer).
```

Reply:
My comment https://github.com/openshift/release/pull/37500#discussion_r1164508883 is still not addressed. :)